src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1058,12 +1089,12 @@ static SvgNode* _createSvgNode(SvgLoaderData* loader, SvgNode* parent, const cha
 static SvgNode* _createMaskNode(SvgLoaderData* loader, SvgNode* parent, const char* buf, unsigned bufLength)
 {
     loader->svgParse->node = _createNode(parent, SvgNodeType::Unknown);
+ 
     if (!loader->svgParse->node) return nullptr;
 
     loader->svgParse->node->display = false;
```

Comment:
If display is false, tvgSceneBuilder cannot apply the style property to the mask.
If you want to use the Alpha value of the mask shape, you need to keep the display (default: true)

Reply:
sure, thank you